win32/tests/threadpool_win32_int/threadpool_win32_int.c

```diff
@@ -36,6 +36,8 @@ typedef struct WAIT_WORK_CONTEXT_TAG
     HANDLE wait_event;
 } WAIT_WORK_CONTEXT;
 
+#define WAIT_WORK_FUNCTION_SLEEP 3000
```

Comment:
>3000 [](http://example.com/codeflow?start=33&length=4)

Consider adding a comment next to this saying // ms
so that we know what we're measuring time in.
Or even better have it in the name of the define: WAIT_WORK_FUNCTION_SLEEP_IN_MS

Reply:
Renamed to `WAIT_WORK_FUNCTION_SLEEP_IN_MS`